content/en/certificates.md

```diff
@@ -3,12 +3,241 @@ title: Chain of Trust
+  * Key type: `RSA 2048`
+  * Validity: expired 2021-10-06
+  * CA details: [crt.sh](https://crt.sh/?caid=16418), [issued certs](https://crt.sh/?Identity=%25&iCAID=16418)
+  * Certificate details (signed by ISRG Root X1): [crt.sh](https://crt.sh/?id=47997543), [der](/certs/letsencryptauthorityx3.der), [pem](/certs/letsencryptauthorityx3.pem), [txt](/certs/letsencryptauthorityx3.txt)
+  * Certificate details (cross-signed by IdenTrust): [crt.sh](https://crt.sh/?id=15706126), [der](/certs/lets-encrypt-x3-cross-signed.der), [pem](/certs/lets-encrypt-x3-cross-signed.pem), [txt](/certs/lets-encrypt-x3-cross-signed.txt)
+* **Let's Encrypt Authority X4**
+  * Subject: `O = Let's Encrypt, CN = Let's Encrypt Authority X4`
+  * Key type: `RSA 2048`
+  * Validity: expired 2021-10-06
+  * CA details: [crt.sh](https://crt.sh/?caid=16429), [issued certs](https://crt.sh/?Identity=%25&iCAID=16429)
+  * Certificate details (signed by ISRG Root X1): [crt.sh](https://crt.sh/?id=47997546), [der](/certs/letsencryptauthorityx4.der), [pem](/certs/letsencryptauthorityx4.pem), [txt](/certs/letsencryptauthorityx4.txt)
+  * Certificate details (cross-signed by IdenTrust): [crt.sh](https://crt.sh/?id=15710291), [der](/certs/lets-encrypt-x4-cross-signed.der), [pem](/certs/lets-encrypt-x4-cross-signed.pem), [txt](/certs/lets-encrypt-x4-cross-signed.txt)
+
+</details>
+
+<details>
+<summary> Delegated OCSP Responder</summary>
+
+This keypair was previously used to sign OCSP responses regarding the status of Let's Encrypt's intermediates on behalf of Let's Encrypt's root, so that the root could remain safely offline. We no longer issue OCSP responses for our intermediates; we instead periodically issue CRLs from our root to convey the revocation status of our intermediates.
+
+* **ISRG Root OCSP X1**
+  * Subject: `O = Internet Security Research Group, CN = ISRG Root OCSP X1`
+  * Key type: `RSA 2048`
+  * Validity: until 2025-06-10
+  * Certificate details (signed by ISRG Root X1): [crt.sh](https://crt.sh/?id=2929281974), [der](/certs/isrg-root-ocsp-x1.der), [pem](/certs/isrg-root-ocsp-x1.pem), [txt](/certs/isrg-root-ocsp-x1.txt)
+  * Certificate details (signed by ISRG Root X1): [crt.sh](https://crt.sh/?id=142051103) (expired)
+
+</details>
+<p><!-- to get the right line spacing after a block element --></p>
+
+# Chains
+
+When an ACME client downloads a newly-issued certificate from Let's Encrypt's ACME API, that certificate comes with a "chain" of one or more intermediates. Usually this chain consists of just the Subscriber certificate itself and the one intermediate which issued it, but it could contain three or more certs, each having issued the previous certificate in the chain. The idea is that, by presenting this whole chain of certificates to a website visitor's browser, the browser will be able to validate the signatures all the way up to a root that browser trusts without having to download any additional intermediates.
```

Comment:
```suggestion
When an ACME client downloads a newly-issued certificate from Let's Encrypt's ACME API, that certificate comes with a "chain" of one or more certificates. Usually this chain consists of just the Subscriber certificate itself and one intermediate, but it could contain additional intermediates. The idea is that, by presenting this whole chain of certificates to a website visitor's browser, the browser will be able to validate the signatures all the way up to a root that browser trusts without having to download any additional intermediates.
```

Explanation:

- The first and second sentence contradicted each other: chain consists of "certificates", not just "intermediates".
- Better to say "certificates" than "certs".
- As of TLS 1.3 the certificates in the chain are not required to certify the previous ones.
https://datatracker.ietf.org/doc/html/rfc8446#section-4.4.2, see "Note: Prior to TLS 1.3"